sources/INSTALL.md

```diff
@@ -1,143 +1,108 @@
-## Build and Installation
-
-  You need OCaml >= 4.04.0, zarith, camlzip, menhir, ocplib-simplex >=
-  0.4 and psmt2-frontend library to compile the sources. You need
-  LablGtk2 and the widget GSourceView2 to compile the GUI. You may
-  need superuser permissions to perform the installation.
-
-#### Configuration
-
-  1. Configure with "./configure" to generate Makefile.config,
+# Build and Installation
+
+## Dependencies
+
+To compile the sources, you will need the following libraries :
+```
+  ocaml >= 4.04.0
+  dune >= 1.5
+  zarith
+  camlzip
+  menhir
+  ocplib-simplex >= 0.4
+  seq
+  cmdliner
+  stdlib-shims
```

Comment:
```suggestion
  stdlib-shims (if ocaml < 4.08)
```
Do we really need stdlib-shims if ocaml >= 4.08 ?

Reply:
This is a transition package, it does nothing if ocaml > 4.08 (https://github.com/ocaml/stdlib-shims/blob/master/src/dune) and it's simplier to let opam handle the constaints over dependencies